Python/crossfire/crossfire/occurrences.py

```diff
@@ -36,14 +40,15 @@ def __next__(self):
         self.yielded += 1
         return occurrence
 
+    @property
+    def url(self):
+        return f"{self.client.URL}/occurrences?{urlencode(self.params, doseq=True)}"
```

Comment:
Amei 💜 

Talvez para legibilidade, dê para quebrar uma etapa:

```suggestion
        params = urlencode(self.params, doseq=True)
        return f"{self.client.URL}/occurrences?{params}"
```

Mas aí já é opinião super subjetiva. Ignore tranquilamente se discordar hehehe : )

Reply:
OK. Aceito a proposta. Entendo que fica mais organizado...